src/applications/registry.json

```diff
@@ -931,15 +931,23 @@
   {
     "appName": "coronavirus-vaccination",
     "entryName": "coronavirus-vaccination",
-    "rootUrl": "/coronavirus-vaccination",
+    "rootUrl": "/covid-19-vaccine/stay-informed",
     "template": {
       "vagovprod": false,
       "layout": "page-react.html",
       "includeBreadcrumbs": true,
       "breadcrumbs_override": [
         {
-          "path": "/coronavirus-vaccination",
-          "name": "Request a COVID-19 vaccination"
+          "path": "/health-care/",
+          "name": "Health care"
+        },
+        {
+          "path": "/health-care/covid-19-vaccine/",
+          "name": "COVID-19 vaccines"
+        },
+        {
+          "path": "/health-care/covid-19-vaccine/stay-informed",
+          "name": "Stay informed and help us prepare"
```

Comment:
URL now starts with `/health-care/` to account for redirect, https://dsva.slack.com/archives/C01FNSTQD7H/p1607460912256200?thread_ts=1607460765.255900&cid=C01FNSTQD7H